.github/workflows/certify-commit.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+# This action test if the last commit of the pull request have been tested on
+# seapath/ansible
+
+name: Certify commit
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  certify-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - run : git clone https://github.com/seapath/ansible.git;
+              cd ansible;
+              git switch debian-main;
+              git submodule update --init -f;
+              cd src/cukinia-tests/;
+              git log --format=format:%H | grep $GITHUB_SHA;
```

Comment:
Did you test this?

Reply:
yes, i test it on my fork with hardcoded GITHUB_SHA.
I don't see any ways to test it directly